types/zen-observable/index.d.ts

```diff
@@ -6,6 +6,8 @@
 //                 BenoitZugmeyer <https://github.com/BenoitZugmeyer>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
+///<reference path="./extras.d.ts"/>
```

Comment:
I don't understand what did you mean here, no reference provided in the description.

You use `zen-observable/extras` in js but `zen-observable` in ts, they are different.

Reply:
Sorry, im still working on my morning coffee.
Interesting this works in the sandbox but is stripped out in our app. I guess this is a project config issue.

Apollogies